src/utilities/view.ts

```diff
@@ -462,10 +463,15 @@ export function createRowSizes({ row1 = '1fr', row2 = '1fr' }: ViewGridRows, col
   return '1fr';
 }
 
-export function validateViewJSONAgainstSchema(json: any) {
+export async function validateViewJSONAgainstSchema(json: any) {
```

Comment:
Does this need to be async? It doesn't look like anything is being awaited in here.

Reply:
Good catch